CI failure in unit tests:

```
>               self.assertFalse(responses)  # all responses should have been consumed
E               AssertionError: [('auth.login', ['XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX']), ('channel.software.listSystemChannels', [[{'channel_name': 'Red Hat Enterprise Linux Server (v. 6 for 64-bit x86_64)', 'channel_label': 'rhel-x86_64-server-6'}]]), ('channel.software.setSystemChannels', [1]), ('auth.logout', [1])] is not false

test/units/modules/packaging/os/test_rhn_register.py:131: AssertionError
```

--- a/lib/ansible/modules/packaging/os/rhn_register.py
+++ b/lib/ansible/modules/packaging/os/rhn_register.py
@@ -306,9 +306,7 @@ class Rhn(redhat.RegistrationBase):
         # Remove systemid file
         os.unlink(self.config['systemIdPath'])
 
-    def subscribe(self, channels=[]):
-        if len(channels) <= 0:
-            return
+    def subscribe(self, channels):
         if self._is_hosted():
             current_channels = self.api('channel.software.listSystemChannels', self.systemid)
             new_channels = [item['channel_label'] for item in current_channels]

